### ISSUE TITLE
Is there an SCP plugin or feature?

### ISSUE THREAD

**Issue by RyannDaGreat** [ranger/ranger#2096]
Is there a way to transfer files over SCP with ranger? (SSH file transfer) Is there perhaps some plugin or feature that would let me do this?

**Comment by toonn** [comment]
You could create a mapping or a rifle rule. For example,
`map xyz console shell scp %f myuser@mymostfrequenthost:~/`.

**Comment by NoSuck** [comment]
Yes, consider rsync if for no other reason than its `cp -n` equivalent that scp sorely lacks (`--ignore-existing`).
